app/classifier/tasks/survey/editor.cjsx

```diff
@@ -247,24 +372,33 @@ module.exports = React.createClass
   makeID: (string) ->
     string.replace(/[aeiouy\W]/gi, '').toUpperCase()
 
-  ensureChoice: (name) ->
+  createChoice: (name) ->
```

Comment:
ensureChoice is a better name because it only creates a choice if that choice did not already exist (not necessary for merge)